frb/data/Galaxies/201124/FRB201124_host.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
I believe this can be removed since the 20201124 version has the full data

Reply:
agreed, removed!